constrain/Classes/AspectRatio.swift

```diff
@@ -12,14 +12,16 @@ public extension Constraints {
     /// Apply an aspect ratio constraint to a view
     /// Ratio is expressed as width/height
     /// Checks that ratio is finite before applying (avoid divide by zero errors)
-    /// Multiplier is not editable, so you need to replace this constraint to change the ratio
-    /// TODO: replacement helper function?
+    /// Multiplier is not editable, so you need to replace this constraint to change the ratio, hence the optional parameter
     @discardableResult
-    func aspectRatio(_ ratio: CGFloat, by relationship: Relationship = .equal, priority: UILayoutPriority = .required) -> Constraints {
+    func aspectRatio(_ ratio: CGFloat, by relationship: Relationship = .equal, priority: UILayoutPriority = .required, replacingExisting: Bool = false) -> Constraints {
```

Comment:
Do you think instead of adding this parameter that it could be the default behavior?

Reply:
It could work, but there are potentially legitimate cases where you could have more than one aspect ratio simultaneously, like different priority and an inequality. And it saves the extra step of removing a nonexistant one in the most common use case where you're not changing it.